sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -160,6 +164,11 @@ def test_non_consecutive_labels():
     assert_almost_equal(ari_1, 0.24, 2)
     assert_almost_equal(ari_2, 0.24, 2)
 
+    ri_1 = rand_score([0, 0, 0, 1, 1, 1], [0, 1, 0, 1, 2, 2])
+    ri_2 = rand_score([0, 0, 0, 1, 1, 1], [0, 4, 0, 4, 2, 2])
+    assert_almost_equal(ri_1, 0.66, 2)
```

Comment:
Yes, this test could likely be generalized and simply compare the results of two identical clusterings that only differ in their label values. On the other hand having an explicit value tests additionally that the routine works as expected (it could theoretically be twice wrong but compute the same values if one test equality on two computations). I have follow the adjusted_rand_score test with my code. I'd suggest to leave this as is and put it up for future restructuring once the rand_index is integrated, as this is a larger issue affecting a more general test setup for several metrics.

Reply:
I parenthesised my comment because it wasn't about your current contribution.

You already test the explicit results, but I think this question of how a metric should behave with a particular anomalous feature of input _should_ be tested through common tests and should _not_ be tested against a be specific result. Again, not your problem